[BUG][BUG]java.lang.Error: java.io.FileNotFoundException: /storage/emulated/0/Pictures/CAP6719366998447182908.jpg (1): open failed: ENOENT (No such file or directory)

**Describe the bug**

code
```
    final List<AssetEntity> assets = await AssetPicker.pickAssets(
      context,
      pickerConfig: AssetPickerConfig(
        maxAssets: CollectionUtils.isNotNull(images) ? 9 - images.length : 9,
        // selectedAssets: pictures,
        themeColor: Theme.of(context).primaryColor,
      ),
    );
```

java.lang.Error: java.io.FileNotFoundException: /storage/emulated/0/Pictures/CAP6719366998447182908.jpg (1): open failed: ENOENT (No such file or directory)

**Flutter version**
flutter 2.10.2
```
  wechat_assets_picker: ^7.0.5
  wechat_camera_picker: ^3.0.0
  photo_manager: ^2.0.1
```
 `compileSdkVersion 32`

android 10

**Log**

error
```
java.lang.Error: java.io.FileNotFoundException: /storage/emulated/0/Pictures/CAP6719366998447182908.jpg (1): open failed: ENOENT (No such file or directory)
E/AndroidRuntime(27669): 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1173)
E/AndroidRuntime(27669): 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
E/AndroidRuntime(27669): 	at java.lang.Thread.run(Thread.java:919)
E/AndroidRuntime(27669): Caused by: java.io.FileNotFoundException: /storage/emulated/0/Pictures/CAP6719366998447182908.jpg (1): open failed: ENOENT (No such file or directory)
E/AndroidRuntime(27669): 	at libcore.io.IoBridge.open(IoBridge.java:496)
E/AndroidRuntime(27669): 	at java.io.FileInputStream.<init>(FileInputStream.java:159)
E/AndroidRuntime(27669): 	at java.io.FileInputStream.<init>(FileInputStream.java:115)
E/AndroidRuntime(27669): 	at androidx.exifinterface.media.ExifInterface.initForFilename(ExifInterface.java:5289)
E/AndroidRuntime(27669): 	at androidx.exifinterface.media.ExifInterface.<init>(ExifInterface.java:3952)
E/AndroidRuntime(27669): 	at com.fluttercandies.photo_manager.core.utils.AndroidQDBUtils.convertCursorToAssetEntity(AndroidQDBUtils.kt:250)
E/AndroidRuntime(27669): 	at com.fluttercandies.photo_manager.core.utils.AndroidQDBUtils.access$convertCursorToAssetEntity(AndroidQDBUtils.kt:29)
E/AndroidRuntime(27669): 	at com.fluttercandies.photo_manager.core.utils.AndroidQDBUtils$getAssetFromGalleryId$1$1.invoke(AndroidQDBUtils.kt:177)
E/AndroidRuntime(27669): 	at com.fluttercandies.photo_manager.core.utils.AndroidQDBUtils$getAssetFromGalleryId$1$1.invoke(AndroidQDBUtils.kt:176)
E/AndroidRuntime(27669): 	at com.fluttercandies.photo_manager.core.utils.AndroidQDBUtils.cursorWithRange(AndroidQDBUtils.kt:138)
E/AndroidRuntime(27669): 	at com.fluttercandies.photo_manager.core.utils.AndroidQDBUtils.getAssetFromGalleryId(AndroidQDBUtils.kt:176)
E/AndroidRuntime(27669): 	at com.fluttercandies.photo_manager.core.PhotoManager.getAssetList(PhotoManager.kt:70)
E/AndroidRuntime(27669): 	at com.fluttercandies.photo_manager.core.PhotoManagerPlugin$onHandlePermissionResult$2.invoke(PhotoManagerPlugin.kt:245)
E/AndroidRuntime(27669): 	at com.fluttercandies.photo_manager.core.PhotoManagerPlugin$onHandlePermissionResult$2.invoke(PhotoManagerPlugin.kt:239)
E/AndroidRuntime(27669): 	at com.fluttercandies.photo_manager.core.PhotoManagerPlugin$Companion.runOnBackground$lambda-0(PhotoManagerPlugin.kt:46)
E/AndroidRuntime(27669): 	at com.fluttercandies.photo_manager.core.PhotoManagerPlugin$Companion.lambda$bZa62ZrXsGssc32bPH4oVAuIci4(Unknown Source:0)
E/AndroidRuntime(27669): 	at com.fluttercandies.photo_manager.core.-$$Lambda$PhotoManagerPlugin$Companion$bZa62ZrXsGssc32bPH4oVAuIci4.run(Unknown Source:2)
E/AndroidRuntime(27669): 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
E/AndroidRuntime(27669): 	... 2 more
E/AndroidRuntime(27669): Caused by: android.system.ErrnoException: open failed: ENOENT (No such file or directory)
E/AndroidRuntime(27669): 	at libcore.io.Linux.open(Native Method)
E/AndroidRuntime(27669): 	at libcore.io.ForwardingOs.open(ForwardingOs.java:167)
E/AndroidRuntime(27669): 	at libcore.io.BlockGuardOs.open(BlockGuardOs.java:252)
E/AndroidRuntime(27669): 	at libcore.io.ForwardingOs.open(ForwardingOs.java:167)
E/AndroidRuntime(27669): 	at android.app.ActivityThread$AndroidOs.open(ActivityThread.java:7461)
E/AndroidRuntime(27669): 	at libcore.io.IoBridge.open(IoBridge.java:482)
E/AndroidRuntime(27669): 	... 19 more
I/Process (27669): Sending signal. PID: 27669 SIG: 9
```

--- a/android/src/main/kotlin/com/fluttercandies/photo_manager/core/utils/AndroidQDBUtils.kt
+++ b/android/src/main/kotlin/com/fluttercandies/photo_manager/core/utils/AndroidQDBUtils.kt
@@ -246,7 +246,7 @@ object AndroidQDBUtils : IDBUtils {
         val modifiedDate = cursor.getLong(MediaStore.MediaColumns.DATE_MODIFIED)
         val orientation: Int = cursor.getInt(MediaStore.MediaColumns.ORIENTATION)
         val relativePath: String = cursor.getString(MediaStore.MediaColumns.RELATIVE_PATH)
-        if ((width == 0 || height == 0) && path.isNotBlank()) {
+        if ((width == 0 || height == 0) && path.isNotBlank() && File(path).exists()) {
             ExifInterface(path).apply {
                 width = getAttribute(ExifInterface.TAG_IMAGE_WIDTH)?.toInt() ?: width
                 height = getAttribute(ExifInterface.TAG_IMAGE_LENGTH)?.toInt() ?: height

--- a/android/src/main/kotlin/com/fluttercandies/photo_manager/core/utils/DBUtils.kt
+++ b/android/src/main/kotlin/com/fluttercandies/photo_manager/core/utils/DBUtils.kt
@@ -255,7 +255,7 @@ object DBUtils : IDBUtils {
         val lng = cursor.getDouble(MediaStore.Images.ImageColumns.LONGITUDE)
         val orientation: Int = cursor.getInt(MediaStore.Images.ImageColumns.ORIENTATION)
         val mimeType = cursor.getString(MediaStore.Files.FileColumns.MIME_TYPE)
-        if ((width == 0 || height == 0) && path.isNotBlank()) {
+        if ((width == 0 || height == 0) && path.isNotBlank() && File(path).exists()) {
             ExifInterface(path).apply {
                 width = getAttribute(ExifInterface.TAG_IMAGE_WIDTH)?.toInt() ?: width
                 height = getAttribute(ExifInterface.TAG_IMAGE_LENGTH)?.toInt() ?: height

--- a/android/src/main/kotlin/com/fluttercandies/photo_manager/core/utils/DBUtils.kt
+++ b/android/src/main/kotlin/com/fluttercandies/photo_manager/core/utils/DBUtils.kt
@@ -311,7 +311,8 @@ object DBUtils : IDBUtils {
 
     override fun getExif(context: Context, id: String): ExifInterface? {
         val asset = getAssetEntity(context, id) ?: return null
-        return ExifInterface(asset.path)
+        val file = File(asset.path)
+        return if (file.exists()) ExifInterface(asset.path) else null
     }
 
     override fun getFilePath(context: Context, id: String, origin: Boolean): String? {

